### PR TITLE
Fixes: The begin value of vertex range in graphscope project fragmet is not always 0

### PR DESCRIPTION
**Pull request by lidongze0629**
## Related issue number

Fixes

### CODE REVIEW

File: grape/utils/vertex_array.h [hunk 1]
```diff
@@ -322,6 +322,10 @@ class DualVertexRange {
     return VertexRange<VID_T>(tail_begin_, tail_end_);
   }
 
+  const VID_T begin_value() const { return head_begin_; }
+
+  const VID_T end_value() const { return tail_end_; }
+
```
Comment (sighingnow) [inline grape/utils/vertex_array.h:328]: Why did you need to expose those two internal fields? Looks not used.

Comment (lidongze0629) [reply grape/utils/vertex_array.h:328]: Use in [sssp](https://github.com/alibaba/libgrape-lite/pull/127/files#diff-310e5cbf407d7b968e25456e8caf45cc692e03d783287ced1f607774eb439c74L158) algorithm with mutable fragment (DualVertexRange)

**Review by sighingnow — APPROVED** [review]
